ui/app/models/pki/role.js

```diff
@@ -84,6 +28,72 @@ export default class PkiRoleModel extends Model {
 
   @attr('string', { readOnly: true }) backend;
 
+  get formFieldGroups() {
+    const defaultArray = [
+      'name',
+      'issuerRef',
+      'customTtl',
+      'notBeforeDuration',
+      'maxTtl',
+      'generateLease',
+      'noStore',
+      'noStoreMetadata',
+      'addBasicConstraints',
+    ];
+    if (this.version.isCommunity) {
+      defaultArray.splice(defaultArray.indexOf('noStoreMetadata'), 1); // remove noStoreMetadata, which is enterprise-only
+    }
```

Comment:
It seems like enterprise only params are becoming more of a theme in PKI - I wonder if making the enterprise attrs an array might be a more extensible way of implementing this logic
```suggestion
    if (this.version.isCommunity) {
     const enterpriseParams = ['noStoreMetadata']
     defaultArray = defaultArray.filter( (attr) => !enterpriseParams.includes(attr))    
    }
```